vng_api_common/notifications/models.py

```diff
@@ -19,9 +18,7 @@ class Meta:
         verbose_name = _("Notificatiescomponentconfiguratie")
 
     def get_auth(self) -> ClientAuth:
-        auth = APICredential.get_auth(
-            self.api_root, scopes=[SCOPE_NOTIFICATIES_PUBLICEREN_LABEL]
-        )
+        auth = APICredential.get_auth(self.api_root, scopes=[])
```

Comment:
```suggestion
        auth = APICredential.get_auth(self.api_root)
```
`scopes` is completely obsolete since a while